jreleaser.yml

```diff
@@ -17,10 +17,17 @@ deploy:
       strict: false
     mavenCentral:
       sonatype:
-        active: ALWAYS
+        active: RELEASE_PRERELEASE
         url: https://central.sonatype.com/api/v1/publisher
         retryDelay: 20
         maxRetries: 100
-        snapshotSupported: true
+        stagingRepositories:
+          - ./repository
+    nexus2:
+      maven-central:
+        active: SNAPSHOT
+        snapshotUrl: https://central.sonatype.com/repository/maven-snapshots/
```

Comment:
You may need the non snapshot URL property as well to be compliant with the config validator. You may set it to the same value as the snapshot URL.

Reply:
Yeap, fixed in https://github.com/quarkiverse/quarkiverse-release/commit/6e9b9324e2977d9eec6effbcfec01bfb64c0ab37, thanks!